blimp-live-widgets/src/components/Countdown/index.js

```diff
@@ -55,41 +100,48 @@ export default class Countdown extends Component {
   startCountdown = () => {
     if (!this.state.countdown && this.state.timeRemaining > 0) {
       this.timer = setInterval(() => {
-        this.decrementTimeRemaining();
-      }, 1000);
-      this.state.countdown = true;
+        this.decrementTimeRemaining()
+      }, 1000)
+      this.state.countdown = true
     }
   };
   stopCountdown = () => {
-    clearInterval(this.timer);
-    this.state.countdown = false;
+    clearInterval(this.timer)
+    this.state.countdown = false
   };
   resetCountdown = () => {
-    this.stopCountdown();
-    this.state.timeRemaining = this.state.timeStart;
-    this.formatTime();
+    this.stopCountdown()
+    this.state.timeRemaining = this.state.timeStart
+    this.formatTime()
   };
   handleFocus = (event) => event.target.select();
 
   render() {
-    const { seconds, minutes, hours } = this.state;
-    let displaySec = ("0" + seconds).slice(-2);
-    let displayMin = ("0" + minutes).slice(-2);
-    let displayHr = ("0" + hours).slice(-2);
+    const { seconds, minutes, hours } = this.state
+    let displaySec = ('0' + seconds).slice(-2)
+    let displayMin = ('0' + minutes).slice(-2)
+    let displayHr = hours.toString()
+    if (displayHr.length < 2) {
+      displayHr = ('0' + hours)
+    }
 
     return (
       <div className={styles.countdown}>
-        <input className={styles.digit} type="text" value={displayHr} onClick={this.handleFocus} onChange={this.changeHours}></input>
-        <span className={styles.colon}>:</span>
-        <input className={styles.digit} type="text" value={displayMin} onClick={this.handleFocus} onChange={this.changeMinutes}></input>
-        <span className={styles.colon}>:</span>
-        <input className={styles.digit} size="1" type="text" value={displaySec} onClick={this.handleFocus} onChange={this.changeSeconds}></input>
-        <br></br>
-
-        <button className={styles.button} onClick={this.startCountdown}> Start </button>
+        <div className={styles.countdownHeader}>
+          {this.state.htnTitle}
+        </div>
+        <div>
+          <input className={styles.digit} type='text' value={displayHr} onClick={this.handleFocus} onChange={this.changeHours} />
+          <span className={styles.colon}>:</span>
+          <input className={styles.digit} type='text' value={displayMin} onClick={this.handleFocus} onChange={this.changeMinutes} />
+          <span className={styles.colon}>:</span>
+          <input className={styles.digit} size='1' type='text' value={displaySec} onClick={this.handleFocus} onChange={this.changeSeconds} />
+          <br />
+        </div>
+        {/* <button className={styles.button} onClick={this.startCountdown}> Start </button>
```

Comment:
Is this just so we can reuse these again later? If yes that's fine, if not we should remove.

Reply:
Yep - to reuse later.